README.md

```diff
@@ -63,6 +63,11 @@ You can either use the provided `Ansible` playbook or run the steps manually
 
     The README uses a plain-text `values.yml` file. Consider using `ansible-vault` as the file includes sensitive information
 
+    4.1. You may optionally set the desired OCP version be setting
+    ```yaml
+    ocp_version: <Your desired OCP version>
```

Comment:
Need also to have option to set this in case using "manual" procedure
This can be done by setting "OCP_VERSION" environment variable, so need to add this to the "manual" instructions in this README